public/index.html

```diff
@@ -7,6 +7,6 @@
   </head>
   <body>
     <div id="root"></div>
-    <script src="index.js" />
+    <script src="index.js"></script>
```

Comment:
Why did you add this line? 🤔 
```suggestion

```

Reply:
Because our coaches told us too. Later it turned out to be irrelevant since react is handling the script command.